android/src/main/java/com/bugsnag/BugsnagReactNative.java

```diff
@@ -175,6 +174,23 @@ public void clearUser() {
     return output;
   }
 
+  private Client getClient(String apiKey) {
+      Client client = null;
+      try {
+          client = Bugsnag.getClient();
+      } catch (IllegalStateException exception) {
+          // Create a new client as none exists yet
+      }
+      if (client == null) {
```

Comment:
we can get rid of the null check by moving this into the catch block

Reply:
Ah true. Totally overlooked that `getClient` is `@Nonnull`